dask/tests/test_base.py

```diff
@@ -677,8 +687,8 @@ def test_compute_dataframe():
     ddf1 = ddf.a + 1
     ddf2 = ddf.a + ddf.b
     out1, out2 = compute(ddf1, ddf2)
-    pd.testing.assert_series_equal(out1, df.a + 1)
```

Comment:
I think the old way was already using `pandas.testing`, right?

Reply:
Yeah I got it backwards originally. Sorry for the mixup.

This just allows these tests to work with older versions of Pandas following [this strategy]( https://github.com/dask/dask/blob/8eeb0e0194ef0561b4202f42de06c0b7fc0784b9/dask/dataframe/_compat.py#L17-L20 ).

Reply:
`pandas.testing` was available in older versions of pandas (including our minimum supported version), so I think we're OK with unconditionally using `pandas.testing`.

Reply:
Is our lower bound on Pandas version sufficient?

https://github.com/dask/dask/blob/283c2bce29f370ee619d651d8ac0cc2ff0f8edb2/setup.py#L20

Should we then get rid of the special casing elsewhere (like this code)?

https://github.com/dask/dask/blob/8eeb0e0194ef0561b4202f42de06c0b7fc0784b9/dask/dataframe/_compat.py#L17-L20

Reply:
I think in that case, we're using an API that wasn't available in 1.0 (`assert_extension_array_equal`).

My high-level question is: what's motivating the change? We test against our minimum supported version, so we should already be OK.

Reply:
We saw some `DeprecationWarning`s cropping up in the UCX-Py. So am trying to address those. I think the Distributed PR ( https://github.com/dask/distributed/pull/4138 ) is probably most relevant, but tried to address things here as well. If the Dask PR is not something we want, I'm ok closing. Though it would be good to get the Distributed PR in if that's agreeable 🙂